File: src/main/java/ski/mashiro/dto/CommentMailDTO.java
package ski.mashiro.dto;

import lombok.Data;

/**
 * @author MashiroT
 */
@Data
public class CommentMailDTO {
    private String articleTitle;
    private String senderNickname;
    private String senderEmail;
    private String receiverNickname;
    private String content;
}

File: mail-service/src/main/kotlin/ski/mashiro/dto/CommentMailDTO.kt
package ski.mashiro.dto

/**
 * @author MashiroT
 */
class CommentMailDTO {
    val articleTitle: String? = null
    val senderNickname: String? = null
    val senderEmail: String? = null
    val receiverNickname: String? = null
    val content: String? = null

    override fun toString(): String {
        return "CommentMailDTO(articleTitle=$articleTitle, senderNickname=$senderNickname, senderEmail=$senderEmail, receiverNickname=$receiverNickname, content=$content)"
    }
}
